### PR TITLE
fix typos, grammar, more concise wording

### PR DESCRIPTION
**Pull request by lolilo**
Signed-off-by: Liana Lo liana.lixia@gmail.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
Saweet.  Thank you. LGTM  You know we are looking for a writer.  HINT

**Comment by lolilo** [comment]
Seems that my pull request was caught in a broken build last night. Should I push again to restart the build, or can the Docker team do this manually? 

As for being a technical writer -- I'm flattered. :) But for now, I'll continue developing my skills as a software engineer. I am very glad to see that Docker has high regard for good docs!

**Comment by moxiegirl** [comment]
@lolilo no worries. I've seen that failure before.  We'll pull this anyway, I just need to do an internal check.

Best of luck with your engineering pursuits.  I'm sure you'll be wonderful.

File: docs/sources/project/set-up-dev-env.md [hunk 1]
```diff
@@ -22,13 +22,12 @@ you continue working with your fork on this branch.
 
 ##  Clean your host of Docker artifacts
 
-Docker developers run the latest stable release of the Docker software; Or 
-Boot2docker and Docker if their machine is Mac OS X. They clean their local
+Docker developers run the latest stable release of the Docker software (with Boot2Docker if their machine is Mac OS X). They clean their local
```
Comment (SvenDowideit) [inline docs/sources/project/set-up-dev-env.md]: 80 char lines please :) (it helps with GitHub's diff view)

**Comment by SvenDowideit** [comment]
LGTM - @lolilo do you have time to make those 2 lines wrapped at 80 chars?

**Comment by moxiegirl** [comment]
@SvenDowideit I can fix the line breaks after the bday. I need to retest the flow after all the tweaking this week.

File: docs/sources/project/set-up-dev-env.md (hunk 1 above)
Comment (lolilo) [inline docs/sources/project/set-up-dev-env.md]: Thanks for the tip! I'll keep this in mind for the future. :)
